hedyorg/hedy
# [CONTENT] Load slides from yaml

Addressed the first part of #4906 by loading code into the slides, rather than loading start codes. I again have learned a valuable lesson here haha. I though "ah I will just half-ass the slides a bit by repurposing something existing, and I will improve it later" and then of course slides are expanded and translated and it gets harder and harder to still improve. I keep forgetting this is not my tiny hobby project anymore :) 

**How to test**
Go to the slides and see that code is loaded from the yaml files.


This also fixes #4794 since it is no longer needed to look up slides code.
